gossip.asciidoc

```diff
@@ -644,68 +644,388 @@ channel graph) has the following attributes:
 ##### Channel Announcement Validation
 
 Now that we know what a `channel_announcement` contains. We can now move onto
-to exactly _how_ to verify such an announcement.
-
+to exactly _how_ to verify such an announcement. Verifying a channel
+announcement entails verifying that:
+
+ 1. The `chain_hash` specified by the channel announcement belongs to a known,
+ and supported blockchain.
+
+ 2. An _unspent_ UTXO exists in the chain as identified by the short channel
+ ID.
+
+ 3. The unspent UTXO has a p2wsh multi-sig script that is composed of
+ `bitcoin_key_1`, and `bitcoin_key_2`, ordered in lexicographic order.
+
+ 4. The signatures of `node_signature_1`, `node_signature_2`,
+ `bitcoin_signature_1`, and `bitcoin_signature_2` are valid signatures under
+ the corresponding public keys over the defined message digest.  The message
+ digest is the serialization of the raw message bytes after the final
+ signature. More on this in the chapter that covers wire message framing.
```

Comment:
I really like including the 4 validation steps into the text. however I wonder if it makes sense if we would actually include the hex of an announcement message, dissect it and run through all steps as an example. It is a lot of work and it seems a bit random to do it for the channel announcement but I guess it is reasonable to do it somewhere and the longer I think about it the more instructive I find it. For example I would not find it that helpful with messages from BOLT 02